apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx

```diff
@@ -159,7 +160,23 @@ const LanguageInformationSection = ({
                 {intl.formatMessage(commonMessages.dividingColon)}
               </span>
               <span data-h2-font-weight="base(700)">
-                {comprehensionLevel}, {writtenLevel}, {verbalLevel}
+                {[
+                  comprehensionLevel
+                    ? intl.formatMessage(
+                        getEvaluatedLanguageAbility(comprehensionLevel),
+                      )
+                    : "",
+                  writtenLevel
+                    ? intl.formatMessage(
+                        getEvaluatedLanguageAbility(writtenLevel),
+                      )
+                    : "",
+                  verbalLevel
+                    ? intl.formatMessage(
+                        getEvaluatedLanguageAbility(verbalLevel),
+                      )
+                    : "",
+                ].join(", ")}
```

Comment:
I beleive this pattern repeats itself a few times. Would it make sense to move it to a single util function?

Reply:
Good call! I've created a new util function for this and updated a few places to use it.